test/e2e/lib/openclaw_tui_issue3145_repro.py

```
@@ -0,0 +1,360 @@
+import struct
+import subprocess
+import sys
+import termios
+import time
+from dataclasses import dataclass
+from typing import Callable
+
+
+ROWS = 80
+COLS = 180
+RAW_LOG = "/tmp/nemoclaw-issue3145-tui-raw.log"
```

Ruff (0.15.13)

[error] Probable insecure usage of temporary file or directory: "/tmp/nemoclaw-issue3145-tui-raw.log"

(S108)